### DISCUSSION TITLE
working on room icons

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#232] [General]
Working on some new icons, but my take a while, I liked my outside, driveway and garden icons enough that I am going to start making a few icons for in home use (for example the outside etc look great on my opening page in my favs room link) so I am adding  (Room icons basically living room, kitchen, bedroom, kids bedroom etc) here is my first one based on kids room

![image](https://user-images.githubusercontent.com/87599807/184077200-15e2d0e3-ad2c-4c06-9081-46729ee2f3af.png)
![image](https://user-images.githubusercontent.com/87599807/184089378-09ccdac5-1859-4063-b07c-4fca29350c02.png)

**Comment by elax46** [comment]
> 

oh really nice ok no rush if you just want to complete the request #228 so then I release at least the version with the latest requests

**Comment by rchiileea** [comment]
![image](https://user-images.githubusercontent.com/87599807/184138305-f69966f1-1918-41fa-9b64-eebda5b4b437.png)
[stage-light-group.zip](https://github.com/elax46/custom-brand-icons/files/9308813/stage-light-group.zip)

I thought it was in other zip lol

**Comment by rchiileea** [comment]
![image](https://user-images.githubusercontent.com/87599807/184138739-bcf55a4b-dadd-4cf8-923a-5c171fa14b9c.png)
![image](https://user-images.githubusercontent.com/87599807/184138779-98d5d030-5f31-4f3c-be0b-c0b83d05ad5a.png)

**Reply by elax46** [reply]
In the dev branch I have already prepared the new section relating to the rooms of the house

**Comment by rchiileea** [comment]
![image](https://user-images.githubusercontent.com/87599807/184532748-ae372b92-baa7-4af6-b7a2-1bac4d4b02ce.png)

**Comment by rchiileea** [comment]
[Rooms.zip](https://github.com/elax46/custom-brand-icons/files/9338982/Rooms.zip)

**Reply by elax46** [reply]
> [Rooms.zip](https://github.com/elax46/custom-brand-icons/files/9338982/Rooms.zip)
> 

I took a few days of rest on my return

**Reply by elax46** [reply]
> [Rooms.zip](https://github.com/elax46/custom-brand-icons/files/9338982/Rooms.zip)

the bathroom icon has multiple paths @rchiileea

**Comment by rchiileea** [comment]
sorry about that, 3 things, fixed bathroom, cut kitchen paths down by 1/3 and new office icon added to this zi
[Rooms-final.zip](https://github.com/elax46/custom-brand-icons/files/9383662/Rooms-final.zip)
p

**Comment by rchiileea** [comment]
all requests done 

[05-09-22.zip](https://github.com/elax46/custom-brand-icons/files/9491654/05-09-22.zip)

**Comment by rchiileea** [comment]
ok no problem, btw also

phu:aqara-opple   shows nothing

phu:aqara-g3    shows nothing

phu:aqara-g2-pro  shows the icon for the phu:aqara-g3
________________________________

**Reply by elax46** [reply]
I checked the errors you reported to me but in my HA everything is displayed correctly
